CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ Changelog is rather internal in nature. See release notes for the public overvie
 
 <!-- Put all new updates into this section, please -->
 
+- [#440]
+  - **Description:** Updates brand theme color tokens and token mapping within components
+  - **Products impact:** Brand token updates
+  - **Addresses:** - https://github.com/learningequality/kolibri-design-system/issues/545
+  - **Components:** -
+  - **Breaking:** - 
```

Comment:
This will be a breaking update as it changes UI significantly

Reply:
I'd also appreciate if the changes could be listed here in a bit more detail, for example what categories of theming were affected. No need to explain which hex color changed into what, rather just to sum up the most important areas of impact.

Reply:
Thank you for this, I was planning to update the changelog with more info but wasn't too sure about the level of detail to include.

Reply:
Thank you, new updates make sense. I see one place that could deserve more detail in my view

When you say
> new content-related token labels have been introduced and existing ones have been renamed.

Could you list the new token labels here, and also before/after labels of existing tokens? So we can find them in the products easily and rename without the need to dig in this PR. I think of this as changes in public API, basically (this is different from just tweaking an existing color - I don't think we need to spend a day on documenting all that).

Reply:
> but wasn't too sure about the level of detail to include.

Generally, you could just imagine that you're someone who's not familiar with the code diff in detail and tried to introduce new theming to Kolibri/Studio. Anything that would be useful for you to know will serve as good guidance :) as I will include it in the release notes. Thank you.